app/src/main/java/com/example/bigowlapp/activity/ListOfScheduleActivity.java

```diff
@@ -25,14 +24,17 @@
 public class ListOfScheduleActivity extends BigOwlActivity {
     private ListView scheduleListView;
     private ScheduleListViewModel scheduleListViewModel;
-    private String groupID, groupName, supervisorName;
+    private String groupID, groupName, supervisorName, supervisorId;
+    private boolean isUserTheGroupSupervisor;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         groupID = getIntent().getStringExtra("groupID");
         groupName = getIntent().getStringExtra("groupName");
         supervisorName = getIntent().getStringExtra("supervisorName");
+        isUserTheGroupSupervisor = getIntent().getBooleanExtra("isUserTheGroupSupervisor", false);
```

Comment:
Good use of intent to reuse an activity instead of making a activity with nearly identical code.